packages/chromatic-api/chromatic-api.js

```diff
@@ -85,26 +86,27 @@ export const Chromatic = {
   addEntry(entry) {
     check(entry, Chromatic.Entry);
     Chromatic._entries[entry.name] = entry;
+    this._entriesDict.set(entry.name, true);
```

Comment:
That would make sense - however entries are full of of live function handles which don't survive `ejson`ing.